demos/browser/app/meeting/meeting.ts

```diff
@@ -474,6 +475,32 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  getNearestMediaRegion(): void {
+    const defaultNearestMediaRegion = "us-east-1";
+    new AsyncScheduler().start(
+      async (): Promise<void> => {
+        this.showProgress('progress-authenticate');
+        try {
+          const nearestMediaRegionResponse = await fetch(
+            `https://nearest-media-region.l.chime.aws`,
+            {
+              method: 'GET',
+            }
+          );
+          const nearestMediaRegionJSON = await nearestMediaRegionResponse.json();
+          const nearestMediaRegion = nearestMediaRegionJSON.region;
+          if(nearestMediaRegion !== null || nearestMediaRegion !== '') {
+            (document.getElementById('inputRegion') as HTMLInputElement).value = nearestMediaRegion;
```

Comment:
We hard-coded the list of available region so you may want to check whether this nearest media region is in that list.

Reply:
If the list of media regions change in the future, the demo app might not reflect the change unless we hard-code and update the list within the demo app.

Reply:
Sure but alternatively, you set a value that is not in the list (UI bug). A better way is to fetch the list of media regions and the nearest media region and populate the list instead of hard coding it.